src/test/java/uk/gov/pay/directdebit/pact/PublicApiContractTest.java

```diff
@@ -136,4 +141,26 @@ public void aDirectDebitEventExists(Map<String, String> params) {
                 .withExternalId(params.getOrDefault("external_id", RandomIdGenerator.newId()))
                 .insert(app.getTestContext().getJdbi());
     }
+
+    @State("a gateway account with external id and a confirmed mandate with a payment on it exists")
+    public void aValidPaymentExists(Map<String, String> params) {
+        testGatewayAccount.withExternalId(params.get("gateway_account_id")).insert(app.getTestContext().getJdbi());
+        testMandate.withGatewayAccountFixture(testGatewayAccount)
+                .withExternalId(MandateExternalId.valueOf(params.get("mandate_id")))
+                .withPayerFixture(PayerFixture.aPayerFixture())
+                .withMandateBankStatementReference(MandateBankStatementReference.valueOf(params.get("bank_mandate_reference")))
```

Comment:
i don't think you need `withMandateBankStatementReference` as it's tangential to testing the `GET`ting of a payment. you'll need to delete `bank_mandate_reference` from the pact in publicapi as well.